### ISSUE TITLE
Intel Graphics Command Center resolution change results in error

### ISSUE THREAD

**Issue by Ferrarisrex** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#828]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Intel Graphics Command Center (beta)

### Processor / Processor Number [Required]

I5-12400F

### Graphic Card [Required]

Intel ARC A750

### GPU Driver Version [Required]

32.0.101.5768

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build [Required]

Windows 10 22H2

### Intel System Support Utility report

[cheese.txt](https://github.com/user-attachments/files/16450309/cheese.txt)

### Description and steps to reproduce [Required]

open Intel Graphics Command Center (Beta)
open "Display" tab
open "General"
under "Resolution" change it to anything
![image](https://github.com/user-attachments/assets/dbda432f-7d4c-4bb5-8122-9f29881096cb)

### Device / Platform

DESKTOP-B6MGHVD

**Comment by Zack-Intel** [comment]
Hi @Ferrarisrex 

Thank you for reporting this. I'll run some tests to try replication and let you know the results once I have more to share.

A few questions that can help clarify the issue:

1. When you say "change it to anything" does it mean native resolutions, custom resolutions, or both?
2. Does this happen on different types of monitors? Do the ports make a difference in this issue?
3. Is this a regression? Or has this happened since the start of using IGCC?
4. Does changing resolution through MS system settings allow changes?

**Comment by Ferrarisrex** [comment]
I can not change to any native resolutions or custom resolutions
it happens on all monitors
I do not know, as i just downloaded IGCC for use
Changing the resolution via windows settings works fine

**Comment by Zack-Intel** [comment]
Thank you @Ferrarisrex 

That's a strange situation. I'm not able to reproduce at this time, but I'll keep testing different configurations and discuss with my team.

In the meantime, the ssu seems to have trouble getting the driver information. This is a sanity check I'm sure you've tried, but can you confirm you tried a clean install of the driver and IGCC beta?

In addition, are there other settings within IGCC that return the same error when trying to configure?

**Comment by Zack-Intel** [comment]
Hi @Ferrarisrex 

Thank you for the update. I know converters to DVI can sometimes cause issues if they're a passive adaptor in not being able to properly display resolutions, but the fact that you can still change the resolutions in Microsoft Display settings is very confusing.

Can you let me know if the resolution changes in Microsoft Display settings get reflected in IGCC once they are changed, as well as what you were able to change to get it working on the Samsung monitor?

**Comment by Ferrarisrex** [comment]
The changes are reflected in IGCC, and then it will let me change the
resolution to a few different resolutions, but only once
The way I was able to get the Samsung monitor to work was messing with the
scale for a while.

Le mer. 7 août 2024, 9:51 AM, Zack-Intel ***@***.***> a
écrit :

**Comment by Zack-Intel** [comment]
Hi @Ferrarisrex 

After some testing, I'm not able to see the issue on my end with a similar configuration. It might be helpful if you can share what resolutions you're trying to get alongside each monitor's edid info, which can be collected in the sys diagnostic report within IGCC.

**Comment by Zack-Intel** [comment]
Hi @Ferrarisrex 

Do you have additional information regarding the edid information to share? If you need help, I can also provide additional instructions on how to collect it

**Comment by Ferrarisrex** [comment]
It seems to have been a problem with the cable, as changing it makes it not
a problem, and putting it back makes it a problem again. This problem
doesn't exist when using this cable with other computers and monitors

Le mer. 4 sept. 2024, 4:16 PM, Zack-Intel ***@***.***> a
écrit :

**Comment by Zack-Intel** [comment]
> It seems to have been a problem with the cable, as changing it makes it not a problem, and putting it back makes it a problem again. This problem doesn't exist when using this cable with other computers and monitors Le mer. 4 sept. 2024, 4:16 PM, Zack-Intel ***@***.***> a écrit :


Thank you for the update. Very strange that it's only that cable and monitor configuration. Sometimes certain adaptors and edids for monitors can clash and lead to corner cases where it doesn't work. I'm glad you were able to resolve this.

I will go ahead and close the ticket. If there are additional issues relating to this, feel free to open a new ticket with the edid information and we can pick up where you left off.
